ClientMVC/Controllers/FridgesController.cs

```diff
@@ -149,5 +149,44 @@ public async Task<ActionResult> UpdateProducts()
 				res = "Error occuped";
 			return View("UpdateProducts", res);
 		}
+
+		[HttpGet]
+		public async Task<ActionResult> AddProductInFridge(Guid fridgeId)
+		{
+			ProductToAddInFridgeViewModel vm = new ProductToAddInFridgeViewModel
+			{
+				FridgeId = fridgeId,
+				Products = await _productService.GetAll()
+			};
+
+			return View(vm);
+		}
+
+		[HttpPost]
+		public async Task<ActionResult> AddProductInFridge(Guid fridgeId, [FromForm] ProductToAddInFridgeDTO productToAdd)
+		{
+			if(productToAdd == null || !ModelState.IsValid)
+			{
+				return View(new ProductToAddInFridgeViewModel
```

Comment:
Your list of products will not change, why do you retrieve data  but if products were added to fridge you don't retrieve data ?
 What does it mean variable res ?